chatmaild/src/chatmaild/tests/plugin.py

```diff
@@ -71,11 +71,11 @@ def maildata(request):
 
     assert datadir.exists(), datadir
 
-    def maildata(name, **kwargs):
+    def maildata(name, from_addr, to_addr, subject="..."):
```

Comment:
shouldn't it be `[...]`?

Reply:
the default before  was  "..."  so i just kept it at that.